### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_RunningTab_ScreenReader: Narrator focus is not in logical order after links present under Notebooks dropdown.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3978]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to run button & hit enter --> Navigate to running tab & hit enter.
3. Start Narrator.
4. Start Navigating through "Notebooks" dropdown using "Caps+Arrow" keys.
5. Verify Narrator focus is not in logical order after "Azure Notebooks - Welcome.ipynb" link.

**Actual:**
1. While navigating using "Caps+Arrow" keys inside "Notebooks" dropdown Narrator focus is not in logical order after links(Azure Notebooks - Welcome.ipynb).
2. After "Azure Notebooks - Welcome.ipynb" link Narrator focus first goes to (Some Plane text where there is no text or controls present then -->Seconds ago then -->Python 2 then-->Shutdown button) which is incorrect. 

**Note:** 
Same Narrator focus order is observed for all the links which are present under "Notebooks" dropdown.

Same issue is observed for "**Chrome+NVDA**" configuration.

**Expected:**
1. While navigating using "Caps+Arrow" keys inside "Notebooks" dropdown Narrator focus should be in logical order after links(Azure Notebooks - Welcome.ipynb) i.e from "Top to Bottom" and from "Left to Right".
2. After "Azure Notebooks - Welcome.ipynb" link Narrator focus Should not go to Some Plane text where there is no text or controls present. 
After "Azure Notebooks - Welcome.ipynb" link Narrator Focus Order should be
 ("Python 2 text"-->"Shutdown button"-->"Seconds ago text").

**UserImpact:**
Screen reader users face difficulty if Narrator focus order is not in logical order.

**Attachment:**
(https://github.com/jupyter/notebook/files/2400265/A11y_AzureNotebooks_Jupyter_RunningTab_ScreenReader.pptx)

**Comment by takluyver** [comment]
Closed by #4740
